.github/workflows/pr-check.yaml

```diff
@@ -137,15 +137,21 @@ jobs:
           echo "Version of @podman-desktop/tests-playwright to be used: $version"
           jq --arg version "$version" '.devDependencies."@podman-desktop/tests-playwright" = $version' package.json > package.json_tmp && mv package.json_tmp package.json
 
-      - name: Execute pnpm in Bootc Extension
+      - name: build extension image
         working-directory: ./podman-desktop-extension-bootc
```

Comment:
issue: As the bot noticed `working-directory` should be changed https://github.com/podman-desktop/extension-bootc/pull/2169#discussion_r2624236929

The option seems to be named `workdir` https://github.com/redhat-actions/buildah-build?tab=readme-ov-file#inputs-for-build-without-containerfile